src/assets/i18n/fr.json

```diff
@@ -362,6 +331,43 @@
 	"Are you sure?": "Êtes vous sûr(e)?",
 	"Do you really want to": "Voulez vous vraiment",
 	"Yes": "Oui",
+	"Share": "Partager",
+	"ITEMS_PER_PAGE_LABEL": "Items par page",
+	"NEXT_PAGE_LABEL": "Page suivante",
+	"PREVIOUS_PAGE_LABEL": "Page précédente",
+	"RANGE_PAGE_LABEL_2": "{{startIndex}} - {{endIndex}} sur {{length}}",
+	"FIRST_PAGE_LABEL": "Première page",
+	"LAST_PAGE_LABEL": "Dernière page",
+	"RANGE_PAGE_LABEL_1": "0 sur {{length}}",
+	"Create new configuration": "Créer une nouvelle configuration",
+	"reduce": "Réduire",
+	"expand": "Déployer",
+	"Hide menu": "Cacher le menu",
+	"New name": "Nouveau nom",
+	"Add a new configuration": "Créer une nouvelle configuration",
+	"Create": "Créer",
+	"Share configuration": "Partage de configuration",
```

Comment:
I would say 'Partager la configuration' to keep using verbs like for 'créer une nouvelle configuration'

Reply:
Fixed